js_src/header.ts

```diff
@@ -0,0 +1,73 @@
+export default class Header {
+    private refersToElement;
+
+    constructor(public element) {
+        HeaderSlideToggle.find(element, "[data-header-slide]");
+    }
+    
+    static apply() {
+        const elements = document.querySelectorAll(".header");
+        let toggles = [];
+        for (let i = 0; i < elements.length; i++) {
+            toggles.push(new Header(elements[i]));
+        }
+        return toggles;
+    }
+};
+
+export class HeaderSlideToggle {
+    private refersToElement;
+
+    public static softDismissDuration = 250; // ms
```

Comment:
see previous comment on variable name

Reply:
better just remove the unneccessary variable... :P